src/overlays/actors/ovl_Bg_Ladder/z_bg_ladder.c

```diff
@@ -8,8 +13,21 @@ void BgLadder_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgLadder_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgLadder_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgLadder_Draw(Actor* thisx, GlobalContext* globalCtx);
+void BgLadder_ActionWait(BgLadder* this, GlobalContext* globalCtx);
+void BgLadder_ActionStartCutscene(BgLadder* this, GlobalContext* globalCtx);
+void BgLadder_ActionFadeIn(BgLadder* this, GlobalContext* globalCtx);
+void BgLadder_ActionIdle(BgLadder* this, GlobalContext* globalCtx);
+
+extern Gfx object_ladder_12rung_dlist[];
+extern Gfx object_ladder_16rung_dlist[];
+extern Gfx object_ladder_20rung_dlist[];
+extern Gfx object_ladder_24rung_dlist[];
+
+extern CollisionHeader object_ladder_12rung_collision;
+extern CollisionHeader object_ladder_16rung_collision;
+extern CollisionHeader object_ladder_20rung_collision;
+extern CollisionHeader object_ladder_24rung_collision;
```

Comment:
I recommend leaving the addresses, at least as a comment. Doing that is way easier to be able to know which symbol is which one when decompiling the objects.

Reply:
Agreed. We are going to be leaving object segment symbols as addresses until we are ready for object reconstruction.